solver/src/http_solver.rs

```diff
@@ -244,6 +247,9 @@ impl HttpSolver {
             tokens: token_models,
             orders: order_models,
             uniswaps: uniswap_models,
+            metadata: Some(MetadataModel {
+                environment: Some(self.network_id.clone()),
```

Comment:
nit: given this is already a string maybe we can translate this into human readable network names (mainnet, rinkeby, xdai)?

Reply:
I thought that network_id was already "mainnet", "rinkeby", etc. Is there a mapping somewhere in the code?

Reply:
If you check where `network_id` is set in the code, you will find that it queries the node for [net_version](https://eth.wiki/json-rpc/API) which returns a string that is a number.

I don't think we have that translation logic (1 => Mainnet, 4=> Rinkeby etc.) yet, but it might be a nice thing to add to the utils crate.

Reply:
naming-nit: Maybe call this `network` instead of `environment`? To me `environment` is more like "production"/"staging".

Reply:
> I don't think we have that translation logic (1 => Mainnet, 4=> Rinkeby etc.) yet, but it might be a nice thing to add to the utils crate.

Sure. Do you mean the "shared" subproject? Or is the "utils" crate on a different repo?

Reply:
Wherever it's convenient in this repo. There is solver/src/utils.rs e.g.